lib/prmd/templates/schemata.md.erb

```diff
@@ -37,6 +38,8 @@ Stability: `<%= schemata['stability'] %>`
 | **<%= name %>** | *<%= type %>* | <%= description %> | <%= example %> |
 <%- end %>
 
+</details>
+
 <%- end %>
 <%- (schemata['links'] || []).each do |link, datum| %>
```

Comment:
From your example, I think there would need to be another details block here also? Is that what you intended?

Reply:
Links means that each link has own details. So, our  code creates documents like this.

```
- Attributes (don't show when there are no properties)
   - Details >
- Link1 (this has anchor for toc link)
  - Details >
- Link2 (also this has.)
  - Details >
```


Currently, Here is link details tag.

https://github.com/uzimith/prmd/blob/fc6e9410b190c1fd3677dece7eaa690e2a891b30/lib/prmd/templates/schemata/link.md.erb#L8

Reply:
Ah, right you are, thanks for clarifying for me.